Azure/sonic-utilities
# Fix show ip/v6 route summary non-multi-asic platform to interact with…

**- What I did**
1. Changed the handling of "show ip/v6 route xxxx" for non-multi-asic platforms to be handled by FRR completely instead of using the common code added to support multi-asic platform.  This is because there are some options such as summary and tables are not yet supported by multi-asic platform and we should not penalize non-multi-asic platform by sharing the same common code.
2. Due to this change, had to modify/add additional unit tests to ensure good test coverage of > 95%.
3. Took the opportunity to refactor the expected test data output strings into a standalone file to make the test code easier to read.

**- How to verify it**
Perform "show ip/v6 route summary" and many other options to ensure that both pizzabox and multi-asic platforms are handling them correctly.
Before the fix:
```
admin@str-a7050-acs-2:~$ show ip route summary
% Unknown command: show ip route sum 
```

After the fix:
```
admin@str-a7050-acs-2:~$ show ip route summary
Route Source         Routes               FIB  (vrf default)
kernel               1                    1                    
connected            26                   26                   
static               1                    0                    
ebgp                 6426                 6425                 
ibgp                 0                    0                    
------
Totals               6454                 6452                 


admin@str-a7050-acs-2:~$
```

This PR is created to port the same change of the master branch to 201911 branch for PR: 
https://github.com/Azure/sonic-utilities/pull/1302
